airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/test/util/destination_process/DestinationProcess.kt

```diff
@@ -60,6 +62,7 @@ abstract class DestinationProcessFactory {
         command: String,
         configContents: String? = null,
         catalog: ConfiguredAirbyteCatalog? = null,
+        useFileTransfer: Boolean,
```

Comment:
Nit: I don't love that this specific feature for s3 is bleeding into the broader interface methods. Don't need to solve now, but we next time we add a specific feature, we'll need to come up with a more extensible pattern.

Reply:
Done. (need to push)